core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -354,6 +400,16 @@ public void configureProperties() throws IOException {
   public abstract OutputStream create(String path) throws IOException;
 
   /**
+   * Creates a file in the under file system with the indicated name and permission status.
+   *
+   * @param path The file name
+   * @param ps the permission status to set for the file
+   * @return A {@code OutputStream} object
+   * @throws IOException if a non-Alluxio error occurs
+   */
+  public abstract OutputStream create(String path, PermissionStatus ps) throws IOException;
```

Comment:
Thanks for the suggestion. Yes, this is for sure better.
I discussed with Calvin earlier and thought there will be a cleanup for UFS API anyways, so I didn't change in this PR. Since you raise it up, I just updated the PR with the change.